### PR TITLE
Added categories and lists

### PR DESCRIPTION
**Pull request by CBID2**
Hi there! I added a Linkedin profile section where She Sharp members can add their links to their profile for networking purposes. I've also added a section for organizations that help women and non-binary people break into Tech. Constructive feedback is greatly appreciated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +34,15 @@ Be sure to read [How To submit your suggestion](./how-to/submit-your-suggestion.
 - [@marwane.dev](https://www.instagram.com/marwane.dev/)
 - [@programmer.nest](https://www.instagram.com/programmer.nest/)
 - [@webapp_creator](https://www.instagram.com/webapp_creator/)
+
+## Linkedin profiles 
+-[Christine Belzie](https://www.linkedin.com/in/christinebelzie)
```
Comment (TessavWalstijn) [inline README.md:39]: Is the idea of Linkedin profiles to add in our community members?
Or more of a place to add groups on linked in that align with SheSharp?

File: README.md [hunk 2]
```diff
@@ -17,10 +17,14 @@ Be sure to read [How To submit your suggestion](./how-to/submit-your-suggestion.
...
-- [Introduction](#introduction)
-  - [Suggest a Resource](#suggesting-a-resource)
+- [SheSharp knowledge hub](#shesharp-knowledge-hub)
+  - [Introduction](#introduction)
+  - [Suggesting a resource](#suggesting-a-resource)
 - [Table of Contents](#table-of-contents)
-  - [Social Media](#social-media)
+- [Social Media](#social-media)
+  - [Instagram profiles](#instagram-profiles)
+  - [Linkedin profiles](#linkedin-profiles)
```
Comment (TessavWalstijn) [inline README.md:26]: We could add Twitter as well, a while back there was some one asking about it.
You are not obligated to include this in any way!

Slack: https://she-sharp.slack.com/archives/CPHESUZ5H/p1664192962620809

**Review by TessavWalstijn — COMMENTED** [review]
Looks good!

File: README.md (hunk 1 above)
Comment (CBID2) [reply README.md:39]: Good question @TessavWalstijn! It's the former.

File: README.md (hunk 2 above)
Comment (CBID2) [reply README.md:26]: Feel free to add that section @TessavWalstijn! :)

**Review by saragerion — APPROVED** [review]
Looks great! Thanks for the PR :D

File: README.md (hunk 2 above)
Comment (TessavWalstijn) [reply README.md:26]: I moved it to a new ticket: https://github.com/shesharpnl/knowledge-hub/issues/4

**Review by TessavWalstijn — APPROVED** [review]

**Pull request merged**